closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -232,23 +240,25 @@ void init(boolean decrypting, String algorithm, byte[] keyValue,
                 keyValue.length + " bytes");
         }
 
-        this.key = keyValue.clone();
-        this.iv = ivValue.clone();
-        this.tagLenBytes = tagLenBytes;
-        this.decrypting = decrypting;
+        synchronized(this) {
+            this.key = keyValue.clone();
+            this.iv = ivValue.clone();
+            this.tagLenBytes = tagLenBytes;
+            this.decrypting = decrypting;
 
-        if (aadBuffer == null) {
-            aadBuffer = new ByteArrayOutputStream();
-        } else {
-            aadBuffer.reset();
-        }
- 
-        if (decrypting) {
-            ibuffer = new ByteArrayOutputStream();
-        }
+            if (aadBuffer == null) {
+                aadBuffer = new ByteArrayOutputStream();
+            } else {
+                aadBuffer.reset();
+            }
+    
```

Comment:
No trailing whitespace, please.

Reply:
Removed.

Reply:
I still see four spaces after `}` (now on line 251).

Reply:
I guess I missed it at some point. But in the latest code, it is gone: [latest revision of the file](https://github.com/ibmruntimes/openj9-openjdk-jdk16/blob/1b0d86c462ac16f44d09414c8b3b6b8061b1154c/closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java#L251)